packages/wonder-blocks-dropdown/src/util/constants.js

```diff
@@ -24,6 +24,7 @@ export const filterableDropdownStyle = {
 export const searchInputStyle = {
     margin: Spacing.xSmall_8,
     marginTop: Spacing.xxxSmall_4,
+    minHeight: "auto",
```

Comment:
Could you also add a comment here to explain why we have to override this please?